tutorials/R/Geospatial-skills/primer-raster-r/Creating-Square-Plot-Boundaries-From-Centroids-in-R/Creating-Square-Plot-Boundaries-From-Centroids-in-R.Rmd

```diff
@@ -110,7 +110,7 @@ library(rgdal)
 
 # read in the NEON plot centroid data 
 # `stringsAsFactors=F` ensures character strings don't import as factors
-centroids <- read.csv("NEON-DS-Field-Site-Spatial-Data/SJER/PlotCentroids/SJERPlotCentroids.csv", stringsAsFactors=FALSE)
+centroids <- read.csv("C:/Users/mccahill/Documents/GitHub/NEON-DS-Field-Site-Spatial-Data/SJER/PlotCentroids/SJERPlotCentroids.csv", stringsAsFactors=FALSE)
```

Comment:
Please change this to the paste0(wd,"NEON-DS-Field...") format (and add a `wd <- "C:/Users/mccahill/Documents/GitHub/"` as needed above)